src/main/java/jalau/at18/katas/bankocr/daniela/BankAccount.java

```diff
@@ -0,0 +1,30 @@
+package jalau.at18.katas.bankocr.daniela;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BankAccount extends Separatedigits {
```

Comment:
BankAccount is a good business logic concept, but not really sure if this relationship (inheritence) with SeparateDigits is correct, maybe should be usage instead? Still, I like that you are trying to leverage inheritance for re-using your code, that is good! keep trying to read your logic in a is-a, can-be and maybe that would help to improve this logic

Reply:
I will review my logic in that part of the code and see if a refactor it's needed and how can I improve that part :)